spec/requests/pages_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+describe "Pages", type: :request do
+  let(:user) { create(:user) }
+
+  describe "#home" do
+    it "respond to request" do
+      get pages_home_path
+      expect(response).to be_successful
+    end
+
+    context "logged in" do
+      before { sign_in user }
+
+      it "has no stories" do
+        list = double
+        expect(Kaminari).to receive(:paginate_array).and_return(list)
+        expect(list).to receive(:page)
+        get pages_home_path
+      end
+
+      it "has stories" do
+        create(:strategy, user_id: user.id)
+        categories = create_list(:category, 2, user_id: user.id)
+        moods = create_list(:mood, 2, user_id: user.id)
+        get pages_home_path
+        expect(assigns(:moment)).to be_a_new(Moment)
+        expect(assigns(:categories)).to eq(categories.reverse)
+        expect(assigns(:moods)).to eq(moods.reverse)
```

Comment:
There might be a better place for some of this test now. Just looking at the response it seems the the categories and moods don't come over. Since we're moving away from controller testing perhaps we should move the parts of this test that covers the categories and moods and only test that the `StoryContainer` and/or the "Test Strategy" are in the response body